### ISSUE TITLE
Unable to run integration tests on ubuntu 20.04 due glibc dependency

### ISSUE THREAD

**Issue by rewenset** [ligato/vpp-agent#1738]
For development I use laptop with Ubuntu 20.04 and when I run `make integration-test` I get an error about `glibc` version:
```
$ make integration-tests 
# running integration tests
VPP_IMG=ligato/vpp-base:20.05 ./tests/integration/vpp_integration.sh
=============================================================
 VPP Integration Test - 20.05.1-5~g09f167997~b57
=============================================================
/vpp-integration.test: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /vpp-integration.test)
-------------------------------------------------------------
 FAILED! (exit code: 1)
-------------------------------------------------------------
make: *** [Makefile:182: integration-tests] Error 1
$
```
I've found that it's happens because binary is compiled in a host with a newer version of glibc:
https://github.com/ligato/vpp-agent/blob/0ead9d2f79c84a9bfe2e54794806c015ded188f3/tests/integration/vpp_integration.sh#L5
And when that binary is executed in a docker container with Ubuntu 18.04 as base it fails with the error.

The same error in TravisCI: https://travis-ci.com/github/rewenset/vpp-agent/builds/188795110

Should I put that compilation process into a container too? What container will be best in that case?
Or maybe there is something else I can do to run integration tests on my laptop?

**Comment by ondrej-fabry** [comment]
This was recently reported and working on a fix with some improvements for e2e test run as well (to make it possible to run in parallel and drop the `--pid=host`).

For a quick fix until this is fixed try compiling like this:

```
env GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go test -ldflags="-s -w" -c ./tests/integration/vpp -o ./tests/integration/vpp/vpp-integration.test 
```

**Comment by rewenset** [comment]
`CGO_ENABLED=0` does not work because of dependency on https://github.com/goccy/go-graphviz in KVScheduler
https://github.com/ligato/vpp-agent/blob/0ead9d2f79c84a9bfe2e54794806c015ded188f3/plugins/kvscheduler/graphviz.go#L17

```
$ make integration-tests
# running integration tests
VPP_IMG=ligato/vpp-base:20.05 ./tests/integration/vpp_integration.sh
# go.ligato.io/vpp-agent/v3/tests/integration/vpp
package go.ligato.io/vpp-agent/v3/tests/integration/vpp (test)
        imports go.ligato.io/vpp-agent/v3/plugins/netalloc/mock
        imports go.ligato.io/vpp-agent/v3/plugins/netalloc
        imports go.ligato.io/vpp-agent/v3/plugins/kvscheduler
        imports github.com/goccy/go-graphviz
        imports github.com/goccy/go-graphviz/cgraph
        imports github.com/goccy/go-graphviz/internal/ccall: build constraints exclude all Go files in /home/andrii/go/pkg/mod/github.com/goccy/go-graphviz@v0.0.6/internal/ccall
FAIL    go.ligato.io/vpp-agent/v3/tests/integration/vpp [setup failed]
FAIL
make: *** [Makefile:182: integration-tests] Error 1
$
```

:persevere:

**Comment by ondrej-fabry** [comment]
Oh right. You could just comment out that section for quick dirty fix.. its used only in one file. I will fix this with build tags with other changes.

**Comment by ondrej-fabry** [comment]
Fixed in #1757
